.travis.yml

```diff
@@ -10,17 +13,12 @@ node_js:
     - "7.8.0"
 
 addons:
-  firefox: "45.0"
+  firefox: "66.0"
```

Comment:
Since we claim backwards compatibility for a while now in our FAQ: https://www.ebi.ac.uk/eva/?Help#supported-browsers, I am not sure how this squares with that. We either change the version here or change the FAQ. Let's have an extended discussion with everyone on this.

Reply:
After discussion, we agreed to move the driver to the most recent browser version so as to have a predictable testing experience. @tskir Could you also please update the FAQ to reflect the new versions?

Reply:
I checked the browser usage in Google Analytics. Over the past month:
- 63% of users use Google Chrome. Of them, **95%** use version 70 or newer; 83% use version 73 (current stable) or newer.
- 19% of users use Mozilla Firefox. Of them, **86%** use version 63 or newer; 75% use version 66 (current stable).

This additionally support moving to Chrome webdriver for testing, as well as focusing more on newest versions of web browsers.